packages/wonder-blocks-toolbar/components/toolbar.js

```diff
@@ -0,0 +1,206 @@
+     * at all.
+     */
+    rightContent?: Array<React.Node>,
+
+    /**
+     * True, if we should show the dismiss icon on the left side o the toolbar.
+     */
+    showClose: boolean,
+
+    /**
+     * How much vertical space to use for the toolbar. If this prop is not
+     * provided, the default is "medium".
+     */
+    size?: "small" | "medium",
+
+    /**
+     * Additional styling for the toolbar. If a color value is provided and
+     * the showClose prop is set to true, the close icon will use this color.
+     */
+    style?: any,
+
+    /**
+     * An optional subtitle rendered in a lighter colour and smaller font size
+     * below the title. Only visible on larger media sizes.
+     */
+    subtitle?: string,
+
+    /**
+     * The main title rendered in larger bold text.
+     */
+    title?: string,
+};
+
+export default class Toolbar extends React.Component<Props> {
+    static defaultProps = {
+        leftContent: [],
+        onClose: () => {},
+        rightContent: [],
+        showClose: false,
+    };
+
+    renderContent(content: React.Node, key: string) {
+        return (
+            <View
+                style={[sharedStyles.content, sharedStyles.verticalAlign]}
+                key={key}
+            >
+                {content}
+            </View>
+        );
+    }
+
+    render() {
+        const {
+            leftContent,
+            onClose,
+            rightContent,
+            showClose,
+            size,
+            style,
+            subtitle,
+            title,
+        } = this.props;
+
+        // TODO(scottgrant): Replace the SVG with an Icon component once
+        // that's done
+        const fillColor = (style && style.color) || "rgba(33, 36, 44, 0.5)";
+        const closeButton = showClose && (
+            <View onClick={onClose}>
+                <svg role="img" width="24px" height="24px">
+                    <path fill={fillColor} d={dismissIcon} />
+                </svg>
+            </View>
+        );
+
+        // TODO(scottgrant): Handle aria-labelledby appropriately.
+        return (
+            <View
+                style={[
+                    sharedStyles.container,
+                    style,
+                    size === "small" && sharedStyles.small,
+                ]}
+            >
+                <View style={sharedStyles.verticalAlign}>
+                    {closeButton}
+                    {leftContent &&
+                        leftContent.map((content, i) =>
+                            this.renderContent(content, i.toString()),
+                        )}
+                </View>
+                <View style={[sharedStyles.column]}>
+                    <View
+                        style={[
+                            sharedStyles.middleColumn,
+                            sharedStyles.verticalAlign,
+                        ]}
+                    >
+                        <span
```

Comment:
Please use components from `wonder-blocks-typography` instead.

Reply:
Oops, thank you! Done.